### ISSUE TITLE
Error after managed blockchain update on April 8th

### ISSUE THREAD

**Issue by isunli** [aws-samples/non-profit-blockchain#10]
Hi,
I tried the code on new version of AWS managed blockchain, and this step is not working:
```
cd ~/non-profit-blockchain/ngo-fabric
source fabric-exports.sh
```
The line:
```
endpoint=$(aws managedblockchain get-node --region $REGION --network-id $NETWORKID --member-id $MEMBERID --node-id $nodeID --query 'Node.Endpoint' --output text)
peerEventPort=$(aws managedblockchain get-node --region $REGION --network-id $NETWORKID --member-id $MEMBERID --node-id $nodeID --query 'Node.FrameworkAttributes.Fabric.PeerEventPort' --output text)
eventEndPoint="${endpoint::-5}$peerEventPort"
```
Give None or empty response.

Thanks!

**Comment by gitesh-tyagi** [comment]
New version of preview has some breaking changes and sample has not been updated yet.

Try:
endpoint=$(aws managedblockchain get-node --region $REGION --network-id $NETWORKID --member-id $MEMBERID --node-id $nodeID --query 'Node.FrameworkAttributes.Fabric.PeerEndpoint' --output text)
eventEndPoint=$(aws managedblockchain get-node --region $REGION --network-id $NETWORKID --member-id $MEMBERID --node-id $nodeID --query 'Node.FrameworkAttributes.Fabric.PeerEventEndpoint' --output text)

**Comment by MCLDG** [comment]
Thanks @isunli and @gitesh-tyagi - I've just merged #11 that should fix this.
